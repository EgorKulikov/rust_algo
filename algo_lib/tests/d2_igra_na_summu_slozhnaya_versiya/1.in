7
3 3 2
2 1 10
6 3 10
6 4 10
100 1 1
4 4 0
69 4 20
