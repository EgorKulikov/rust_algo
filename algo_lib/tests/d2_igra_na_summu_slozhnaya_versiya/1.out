6
5
375000012
500000026
958557139
0
49735962
